package.json

```diff
@@ -15,7 +15,9 @@
     "build": "tsdx build",
     "test": "tsdx test",
     "test:coverage": "tsdx test --coverage",
-    "lint": "tsdx lint src test",
+    "lint": "run-p lint:*",
+    "lint:eslint": "tsdx lint src test",
+    "lint:typescript": "tsc --noEmit",
```

Comment:
Added the "tsc --noEmit" for compiler errors, then run both in parallel with `run-p`